templates/web/search.ftl

```diff
@@ -124,6 +124,25 @@
                                 </div>
                             </div>
 
+                            <div class="panel panel-default">
+                              <div class="panel-heading" role="tab" id="headingPath">
```

Comment:
is this id repeated?

Reply:
no it isn't, is the id used for this new filter